### ISSUE TITLE
Email notifications are incorrectly showing Entries fields

### ISSUE THREAD

**Issue by lindseydiloreto** [verbb/formie#1118]
### Describe the bug

When a form with an "Entries" field type is submitted, the notification email is incorrectly displaying the results.

---

## Email Results

<img width="240" src="https://user-images.githubusercontent.com/5309692/195219788-feec0908-b4b5-4a98-b220-41f8020d028f.png" alt="">

## How it appears in the form

<img width="800" src="https://user-images.githubusercontent.com/5309692/195219887-c1a38a64-0193-4ec8-8569-5cd6911cfa75.png" alt="">

## Defined as an "Entries" field type

<img width="400" src="https://user-images.githubusercontent.com/5309692/195219929-749f4405-d7d1-4740-b9c3-e8884b6d63c2.png" alt="">

---

Let me know if you need additional information!

### Steps to reproduce

1. Use an "Entries" field.
2. Create a notification email.
3. Trigger a message to yourself.

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.2.5.2

### Plugin version

2.0.12

### Multi-site?

No

**Comment by engram-design** [comment]
Can you confirm the value for the submission for the entries field? Have you selected an entry, or just left blank?

Otherwise, I can't replicate that, as leaving empty shows "No response." and selecting an entry shows just the selected entry.

You're not using any email template overrides, or anything?

**Comment by lindseydiloreto** [comment]
Good question, it was a product of the test email (at the bottom of the preview window).

I'm not sure what happens if you submit the form normally, I can test it out if you'd like.

**Comment by engram-design** [comment]
Oh right, I had a feeling this might've been using the preview functionality. That's expected behaviour, as it's just showing what it'll look like, as it's got no value to work from.

**Comment by lindseydiloreto** [comment]
Interesting, wouldn't it just choose a value at random? That's what the other dropdowns do.

Regardless, it's good to hear that this only happens with the test email! Feel free to close this thread if you want. 🙂 

Thanks for helping get to the bottom of that!

**Comment by engram-design** [comment]
It probably should! (Note is already respects the limit setting to show a single entry if set). Should be fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.13](https://github.com/verbb/formie/releases/tag/2.0.13)
